client/src/components/PastaShow.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react'
+
+const PastaShow = (props) => {
+    const [pasta, setPasta] = useState({
+        name: '',
+        description: '',
+        category: {}
+    })
+
+    const pastaId = props.match.params.id
```

Comment:
This could be defined within the getPasta function since that's the only place it's used